doc/source/tune/_tutorials/tune-60-seconds.rst

```diff
@@ -146,19 +151,24 @@ Trial Schedulers are Tune modules that adjust and change distributed training ru
 
     # Execute 20 trials using HyperBand using a search space
     configs = {"lr": tune.uniform(0, 1), "momentum": tune.uniform(0, 1)}
-    tune.run(MyTrainableClass, num_samples=20, config=configs, scheduler=hyperband)
+
+    tune.run(
+        MyTrainableClass,
+        config=configs,
+        num_samples=20,
+        scheduler=hyperband
+    )
+
+Population-based training and HyperBand are examples of popular optimization algorithms implemented as Trial Schedulers.
 
 Unlike **Search Algorithms**, Trial Schedulers do not select which hyperparameter configurations to evaluate. However, you can use them together.
 
 See the documentation: :ref:`schedulers-ref`.
```

Comment:
I always prefer inline links as opposed to end statement links. Also, there's no harm in linking multiple times, some inline and some at the end.

Reply:
done